pkgs/applications/misc/terminus/default.nix

```diff
@@ -28,16 +88,22 @@ stdenv.mkDerivation rec {
     mkdir -p "$out/bin"
     mv opt "$out/"
     ln -s "$out/opt/Terminus/terminus" "$out/bin/terminus"
-    patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" --set-rpath "${libPath}:\$ORIGIN" "$out/opt/Terminus/terminus"
+
+    patchelf --set-interpreter ${stdenv.cc.bintools.dynamicLinker} \
+        "$out/opt/Terminus/terminus"
+
+    wrapProgram $out/bin/terminus \
+        "''${gappsWrapperArgs[@]}" \
+        --prefix XDG_DATA_DIRS : "${gtk3}/share/gsettings-schemas/${gtk3.name}/" \
+        --prefix LD_LIBRARY_PATH : ${libPath}
     mv usr/* "$out/"
-    wrapProgram $out/bin/terminus --prefix PATH : ${lib.makeBinPath [ nodejs ]}
   '';
   dontPatchELF = true;
   meta = with lib; {
     description = "A terminal for a more modern age";
-    homepage    = "https://eugeny.github.io/terminus/";
-    maintainers = with maintainers; [ jlesquembre ];
-    license     = licenses.mit;
-    platforms   = [ "x86_64-linux" ];
+    homepage = "https://eugeny.github.io/terminus/";
+    maintainers = with maintainers; [];
```

Comment:
Please add yourself back as a maintainer.

Reply:
Related: https://github.com/NixOS/nixpkgs/issues/85232
I tried to fix the derivation, but it didn't work. I'm removing myself as maintainer because I don't use terminus anymore, and I don't have time to work on this.